anydex/test/wallet/ethereum/token/abi.json

```diff
@@ -0,0 +1,222 @@
+[

```

Comment:
This file seems to be repeated three times. Could a function that creates a new one where needed be implemented, similar to how bitcoinlib wallets create config files, instead? Or is this necessary for a different reason?

Reply:
It's mainly repeated due to testing.